crates/reth/primitives/src/lib.rs

```diff
@@ -22,6 +23,7 @@ sol! {
         /// Withdrawal amount in sats
         uint64 amount,
         /// 32 bytes pubkey for withdrawal address in L1
-        bytes32 dest_pk,
+        // FIXME: How do I use a BOSD Descriptor here?
+        bytes32 destination,
```

Comment:
`bytes`, unfortunately

This kinda still is messy, we shouldn't be using logs for this imo.

Reply:
Done with `byte` thanks!